packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -2212,8 +2211,10 @@ describe("Teal Opcodes", function () {
         op = new AppLocalGetEx([], 1, interpreter);
         op.execute(stack);
 
-        top = stack.pop();
-        assert.deepEqual(toBytes('Local-val'), top);
+        flag = stack.pop();
+        value = stack.pop();
+        assert.deepEqual(BigInt("1"), flag);
```

Comment:
BigInt provides a native comparison operator, so let's use `assert.equal`

Reply:
plrsdr update below as well, in places where we use BigInt

Reply:
ok